### PR TITLE
Shooting methods and Vertical checking

### PR DESCRIPTION
**Pull request by ghost**
Added vertical variable to ship class in order to make updateShot() method simpler, as well as a method for checking vertical on ship class. 

fireAt() method of main is now (mostly) functional. Method calls updateShot() on Battleshipmodel, checks whether the shot's coordinates coincide with and ship locations. Upon hit, it will break out of it's loop and update the playerHits, or computerHits array with respect to who's shooting. updateShot() also returns a boolean, true if hit, if we would like to double check on the main method.

Wrote BattleshipModelTest and created four tests for the class. All currently pass. Also began to write tests for fireAt() method of main, still working on those.

### CODE REVIEW

File: src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java [hunk 1]
```diff
@@ -1,17 +1,21 @@
 package edu.oregonstate.cs361.battleship;
 
+import com.sun.javafx.image.BytePixelSetter;
```
Comment (TravisWhitehead) [inline src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java]: This looks like a typo.

**Review by TravisWhitehead — APPROVED** [review]
Looks like there's an accidental import in there. Some of the variable names (like isVert) aren't my favorite, but that's a non-issue. The updateShot method is gonna be useful.

File: src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java (hunk 1 above)
Comment (ghost) [reply src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java]: That's interesting. Apparently IntelliJ automatically adds imports to things that are typos. Adding editing the pull request. 

Please hold.

Comment (TravisWhitehead) [reply src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java]: Looks good.

**Comment by ghost** [comment]
Alright, should be good to go.

**Pull request merged**